src/System.IO.Pipelines/src/System/IO/Pipelines/PipeWriter.cs

```diff
@@ -48,10 +48,20 @@ public abstract partial class PipeWriter : IBufferWriter<byte>
         /// <summary>
         /// Returns a <see cref="Stream"/> that wraps the <see cref="PipeWriter"/>.
```

Comment:
More borrowing of docs

```suggestion
         /// Returns a <see cref="Stream"/> that wraps the <see cref="PipeWriter"/>, which optionally leaves the pipe open when disposed.
```